tutorial/managing-software.rst

```diff
@@ -0,0 +1,1611 @@
+part of the original LTS release, and so it takes the name of the Ubuntu series
+by default (i.e., the ``-release`` pocket is implied). 
+
+Components
+^^^^^^^^^^
+
+Each pocket is split into four **components**, depending on whether the packages
+they contain are *open source* or *closed source*, and whether they are
+officially supported by Canonical or are maintained by the Ubuntu Community:
+
++--------------------------+-------------+---------------+
+|                          | Open source | Closed source |
++==========================+=============+===============+
+| **Officially supported** | main        | restricted    |
++--------------------------+-------------+---------------+
+| **Community supported**  | universe    | multiverse    |
++--------------------------+-------------+---------------+
+
+- **main** contains the open-source packages that are officially supported by
+  Canonical. These packages are either installed on every Ubuntu machine, or
+  are very widely used for various types of systems.
+- **universe** holds all other open-source packages in Ubuntu, which are
+  typically maintained by the Debian and Ubuntu communities, but may also
+  include additional security coverage from Canonical under
+  `Ubuntu Pro <https://ubuntu.com/pro>`_, which is available free for personal
+  use on up to five machines.
+- **restricted** contains the packages that are officially supported by
+  Canonical but are not available under a completely free license.
+- **multiverse** contains community-maintained proprietary software -- these
+  packages are completely unsupported by Canonical.
+
+If you would like more information about the Ubuntu release process, how
+packages are produced, or to learn more about the sort of terminology you might
+come across, you may be interested in the
+`Ubuntu Packaging Guide <https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/explanation/archive/>`_,
+which is a great resource containing all this information (and much more!).
+
+(Optional) building from source
+===============================
+
+.. note::
+    This next section is a bit more of a challenge, so if you're happy with
+    what we've covered so far and want to :ref:`skip on ahead <tutorial_snaps>`,
+    you can! Otherwise, read on...
+
+Although APT is the preferred way to install packages on your system, due to
+its ability to handle depedencies and keep software up-to-date, not every
+package is available in the APT repository -- especially if they are so old
+they are no longer maintained, or conversely, are the newest version still in
+development!
+
+We can handle deb packages that aren't in the APT repository using ``dpkg`` --
+all we need is to download the deb, and we can run a command like this to
+install it:
+
+.. code-block:: bash
+
+    sudo dpkg -i <package name>
+
+However, it is also possible to obtain software by building it from the source.
```

Comment:
Furthermore this whole topic is so very different based on which software you talk about - that none of the instructions might match when someone does something else. So even when moving elsewhere, you should add even more intense "this will be very different depending on what you do, this is just one example"

And again, not good to be here in the tutorial.

Makes the otherwise great "how you should use it" complicated end ends with showing something people should better not use - at least not until they really++ know what they do.

Moving installing from manually downloaded .deb
And building source to separate pages each and linking from the tutorial as "there is more, but you should almost never need that and it comes with drawbacks to maintainability of the systems (no updates, ...)" is what I'd expect.

Reply:
Removed this entire section, as discussed